### PR TITLE
Add approve tokens test scenarios

### LINKED ISSUE #18943: Approval: ERC20 Token approvals

### LINKED ISSUE #18944: Approval: ERC721 and ERC1155 token approvals

### PR DESCRIPTION
**Pull request by chloeYue**
## **Description**
Add new test scenarios about approving tokens for manual testing.
Also adjusting some test scenarios repo name.

## **Related issues**

Fixes: #18943 and #18944 

## **Manual testing steps**

Not necessary.

## **Pre-merge author checklist**

- [X] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [X] I've clearly explained what problem this PR is solving and how it is solved.
- [X] I've linked related issues
- [X] I've included manual testing steps
- [X] I've included screenshots/recordings if applicable
- [X] I’ve included tests if applicable
- [X] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [X] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [X] I’ve properly set the pull request status:
  - [X] In case it's not yet "ready for review", I've set it to "draft".
  - [X] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [X] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [X] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by DDDDDanica — DISMISSED** [review]

File: test/scenarios/4. tokens/approve erc721 token.csv [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+9,Switch to the wallet and open the activity list item.,,"The contract deployment transaction status, nonce, amount, gas and total are shown in the item details.",
+10,"Proceed to mint NFT. ",,,
+11,Confirm the transaction and wait for it to complete.,,,
+12,Switch to the wallet and open the activity list item.,,"The mint NFT transaction status, recipient's address, nonce, amount, gas and total are shown in the item details.",
+13,Proceed to view the transaction on the block explorer.,,The block explorer opens in a new tab.,
+14,"Switch to test dapp, in the input field of approve NFT token section, enter the NFT ID you wish to approve.",e.g. 3,,"If NFT with token ID #3 should be appproved, amount of NFTs minted should be equal to or greater than 3."
+15,"In the test dapp, click ""Approve"".",,"Notification window is shown, asking user to allow access to the minted NFT.",
+16,Proceed to confirm the NFT approval.,,,
+17,Switch back to the wallet.,,The approve NFT spending cap transaction appears in the activity list. The recipient's hexadecimal address is shown in the activity list item. The amount is shown in the activity list item.,
```
Comment (PeterYinusa) [inline test/scenarios/4. tokens/approve erc721 token.csv:18]: > The recipient's hexadecimal address is shown in the activity list item. The amount is shown in the activity list item.

This is not shown on Approve spending cap list item

File: test/scenarios/4. tokens/approve erc20 token.csv [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+4,Open the test dapp in another tab.,https://metamask.github.io/test-dapp/,,
+5,"Proceed to connect with MetaMask. ",,,
+6,"Connect with the current account. ",,,
+7,Deploy the token contract.,,,
+8,Confirm the transaction and wait for it to complete.,,,
+9,Switch to the wallet and open the activity list item.,,"The contract deployment transaction status, nonce, amount, gas and total are shown in the item details.",
+10,Proceed to view the transaction on the block explorer.,,The block explorer opens in a new tab.,
+11,"In the test dapp, proceed to approve tokens. ",,"Spending cap request notification window is shown. The token icon and symbol TST are shown. An input field is displayed, allow user to edit the spending cap amount .",
+12,Proceed to approve the spending cap request.,,,
+13,Switch back to the wallet.,,"The transaction ""Approve TST spending cap"" appears in the activity list. The recipient's hexadecimal address is shown in the activity list item. The amount is shown in the activity list item.",
```
Comment (PeterYinusa) [inline test/scenarios/4. tokens/approve erc20 token.csv:14]: Same as below

File: test/scenarios/4. tokens/approve erc721 token.csv (hunk 1 above)
Comment (chloeYue) [reply test/scenarios/4. tokens/approve erc721 token.csv:18]: Nice catch, thanks !  Corrected.

File: test/scenarios/4. tokens/approve erc 1155 token.csv [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+7,Deploy the ERC1155 contract.,,,
+8,Confirm the transaction and wait for it to complete.,,,
+9,Switch to the wallet and open the activity list item.,,"The contract deployment transaction status, nonce, amount, gas and total are shown in the item details.",
+10,Proceed to Batch Mint tokens.,,,
+11,Confirm the transaction and wait for it to complete.,,,
+12,Switch to the wallet and open the activity list item.,,"The batch mint token transaction status, recipient's address, nonce, amount, gas and total are shown in the item details.",
+13,"Switch to test dapp, click ""Set approval for all"" in ERC1155 token section.",,"Notification window is shown, asking user to allow access to all the minted ERC1155 tokens.",
+14,Proceed to confirm the tokens approval.,,"A prompt warning message ""Your NFT may be at risk"" is shown.",
+15,"Continue to confirm tokens approval. ",,,
+16,Switch back to the wallet.,,"The ""Approve token with no spend limit"" transaction appears in the activity list. The recipient's hexadecimal address is shown in the activity list item. The amount is shown in the activity list item.",
```
[the remark this answers is not in the document]
Comment (chloeYue) [reply test/scenarios/4. tokens/approve erc 1155 token.csv:21]: Corrected. Thanks!

**Review by darkwing — DISMISSED** [review]

**Review by pedronfigueiredo — APPROVED** [review]

**Review by benjisclowder — COMMENTED** [review]
Looks great!

**Review by benjisclowder — APPROVED** [review]

**Pull request merged**